playgrounds/vue/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Logs
```

Comment:
Minor suggestion.

Do we need separate ```.gitignore``` in the Vue playground project?
It looks like it was auto-generated
Maybe we should delete it?

Reply:
done